portfolio/src/main/java/com/google/sps/servlets/GetCommentsServlet.java

```diff
@@ -31,15 +31,15 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/comments")
-public class DataServlet extends HttpServlet {
+/** Servlet that returns comments data */
+@WebServlet("/get-comments")
+public class GetCommentsServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
FYI, generally it's pretty common/ok to have 1 endpoint that takes GET and POST requests. E.g. /comments.

But this is fine too.